bitbots_localization/src/localization.cpp

```diff
@@ -361,50 +363,77 @@ void Localization::getMotion() {
 void Localization::publish_transforms() {
 
   //get estimate and covariance
-  estimate_ = robot_pf_->getBestXPercentEstimate(config_->percentage_best_particles);
-  estimate_cov_ = robot_pf_->getCovariance(config_->percentage_best_particles);
+  auto estimate = robot_pf_->getBestXPercentEstimate(config_->percentage_best_particles);
 
-  //calculate quaternion
+  // Convert to tf2
+  tf2::Transform filter_transform;
+  filter_transform.setOrigin(tf2::Vector3(estimate.getXPos() , estimate.getYPos(), 0.0));
   tf2::Quaternion q;
-  q.setRPY(0, 0, estimate_.getTheta());
-  q.normalize();
+  q.setRPY(0, 0, estimate.getTheta());
+  filter_transform.setRotation(q);
+
+  // Get the transform from the last measurement timestamp until now
+  geometry_msgs::msg::TransformStamped odomDuringMeasurement, odomNow;
```

Comment:
All of the other variables are in snake_case, so I would prefer to do the same here.

Reply:
Other transforms are also in camel case.
Our future linter will complain soon(TM)